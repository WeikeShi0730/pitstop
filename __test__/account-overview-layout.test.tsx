import { render, RenderResult } from "@testing-library/react";
import AccountOverviewLayout from "../components/account-overview-layout.component";

let documentBody: RenderResult;
describe("<AccountOverviewLayout />", () => {
  beforeEach(() => {
    documentBody = render(
      <AccountOverviewLayout>
        <></>
      </AccountOverviewLayout>
    );
  });
  it("shows content in <AccountOverviewLayout />", () => {
    const { baseElement } = documentBody;
    expect(baseElement).toMatchSnapshot();
  });
});
