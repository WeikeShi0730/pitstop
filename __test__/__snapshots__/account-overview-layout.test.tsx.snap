// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AccountOverviewLayout /> shows content in <AccountOverviewLayout /> 1`] = `
<body>
  <div>
    <div
      class="w-full max-w-xs sm:max-w-sm md:max-w-md lg:max-w-lg m-auto"
    >
      <div
        class="text-slate-700 bg-slate-100 bg-opacity-30 rounded-lg py-5 px-5 shadow-md hover:shadow-slate-500 transition-all ease-in-out duration-200"
      >
        <div
          class="flex flex-col items-center"
        />
      </div>
    </div>
  </div>
</body>
`;
